packages/main/src/themes/sap_horizon/rtl-parameters.css

```diff
@@ -0,0 +1,6 @@
+@import "../base/rtl-parameters.css";
+
+[dir="rtl"] {
```

Comment:
Do we need this? As these are only parameters, or the loading of the variables here also depend on that attribute?

Reply:
Same for every rtl file.

Reply:
All parameters imported inside parameters-bundle.css are bundled inside one large css files that contains all the css variables for the respective theme.

The file has following structure: 

```css
:root {}

[dir="rtl"] {}
```

So rtl variables are always loaded with other theme parameters but they only apply when the selector is matched.